flatland/__main__.py

```diff
@@ -57,6 +58,10 @@ def parse(cl_input):
                         help='Rebuild the flatland database. Necessary only if corrupted.')
     parser.add_argument('-COLORS', '--colors', action='store_true',
                         help='Show the list of background color names')
+    parser.add_argument('-micca', '--micca', action='store', default='domain',
```

Comment:
I think the convention for "-" is a short (1 or 2 character) option/argument while "--" is for longer ones such as "--micca".